tests/suite/model/numbering.typ

```diff
@@ -24,6 +24,25 @@
   [ for #i \ ]
 }
 
+--- numbering-greek ---
```

Comment:
New numbering tests should not use reference images anymore, but rather assertions. The old ones should also be updated some time.

Reply:
Update the test case, now only assertions, not sure whether it is what you want it to be. If so, do you want all the numbering test to be change to assertions in the numbering.type file like this?

(I previously thought that the typ and ref png should be the test on rendering results, and because of so for the numbering I added some local test to `numbering.rs` where the function is defined.)

Reply:
>  If so, do you want all the numbering test to be change to assertions in the numbering.type file like this?

In principle yes, but that can be done separately. But open a PR if you'd like!

> I previously thought that the typ and ref png should be the test on rendering results, and because of so for the numbering I added some local test to numbering.rs where the function is defined.

I think the tests should just be where it's most convenient. If it's easier to test in Typst code, test it there, if it's simpler to test in Rust, test it there. This applies to the logic itself. The interface must be tested in Typst code to ensure that it works in integration.